src/hourly-weather.ts

```diff
@@ -82,6 +84,9 @@ export class HourlyWeatherCard extends LitElement {
   private _labels = LABELS;
   private labelsLocalized = false;
 
+  private _directions = Object.keys(DIRECTIONS);
+  private directionsLocalized = false;
```

Comment:
Make sure to set to `false` after line 96 just like we do for the `labelsLocalized` variable.